Library/Homebrew/tap.rb

```diff
@@ -673,8 +673,13 @@ def new_cask_path(token)
 
   sig { params(token: String).returns(String) }
   def relative_cask_path(token)
-    new_cask_path(token).to_s
-                        .delete_prefix("#{path}/")
+    if token.start_with?("font-")
```

Comment:
It seems simpler to me to move the `token.start_with?` logic into `new_cask_path` instead of adjusting the callers in various places.